client/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "client",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "react": "^16.5.0",
+    "react-dom": "^16.5.0",
+    "react-scripts": "1.1.5"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test --env=jsdom",
+    "eject": "react-scripts eject"
+  },
+  "proxy": "http://localhost:3000"
```

Comment:
I'm not sure this is correct. Here is an example of how I used this in previous projects:

```
"proxy": {
    "/*": {
      "target": "http://localhost:3000"
     }
  }

Reply:
we tried it, and it's working.
if any changes happens i will use yours. thanks